src/providers/ContentShareProvider/docs/useContentShareState.mdx

```diff
@@ -57,6 +75,167 @@ const MyChild = () => {
 };
 ```
 
+### Single Content Share with All Properties
+
+```jsx
+import React from 'react';
+import {
+  MeetingProvider,
+  useContentShareState,
+  ContentShare,
+} from 'amazon-chime-sdk-component-library-react';
+
+const App = () => (
+  <MeetingProvider>
+    <ContentShareView />
+  </MeetingProvider>
+);
+
+const ContentShareView = () => {
+  const {
+    // Deprecated properties (maintained for backward compatibility)
+    tileId,
+    sharingAttendeeId,
+
+    // Content share state
+    paused,
+    isLocalUserSharing,
+    isLocalShareLoading,
+    canStartContentShare,
+
+    // Collections for multiple content shares
+    tiles,
+    tileIdToAttendeeId,
+    attendeeIdToTileId
+  } = useContentShareState();
+
+  // No content is being shared
+  if (tiles.length === 0) {
+    return (
+      <div className="content-share-container">
+        <p>No content is being shared</p>
+        <p>Can start content share: {canStartContentShare ? 'Yes' : 'No'}</p>
+      </div>
+    );
+  }
+
+  // Content is being shared
+  const contentShareTileId = tiles[0] // Or deprecated `tileId`
+  const attendeeId = attendeeIdToTileId[contentShareTileId.toString()] // Or deprecated `sharingAttendeeId`
```

Comment:
Can the TileId's be `number`s and the key in `attendeeIdToTileId` be `number` to avoid this `toString()` explicit conversions?

Reply:
Yeah object key can be a number. But this how RemoteVideoTileProvider is doing it, I decide to follow the same.